include/swift/AST/KnownSDKTypes.def

```diff
@@ -40,6 +40,7 @@ KNOWN_SDK_TYPE_DECL(ObjectiveC, ObjCBool, StructDecl, 0)
 KNOWN_SDK_TYPE_DECL(Concurrency, CheckedContinuation, NominalTypeDecl, 2)
 KNOWN_SDK_TYPE_DECL(Concurrency, UnsafeContinuation, NominalTypeDecl, 2)
 KNOWN_SDK_TYPE_DECL(Concurrency, MainActor, NominalTypeDecl, 0)
+KNOWN_SDK_TYPE_DECL(Concurrency, nonisolated, NominalTypeDecl, 0)
```

Comment:
Hmm, I don't think we should be pulling such sneaky tricks. It's adding more confusion to an already complex world of keywords, attributes and now there'd be a type with the same name as a keyword as well but ofc not interchangable. Let's acknnowladge it's a type and follow usual Swift naming conventions, so `Nonisolated` for this

Reply:
This line of code isn't doing anything other than generating some compiler API to easily get a known type in the Concurrency library. The actual part of this I think you object to is the type I added to `stdlib/public/Concurrency/Actor.swift`:

```swift
@available(SwiftStdlib 6.2, *)
public struct nonisolated {}
```

But yes, let's discuss this on the forum thread because that's design feedback.